android/expoview/src/main/java/host/exp/exponent/exceptions/ManifestException.kt

```diff
@@ -50,7 +50,7 @@ class ManifestException : ExponentException {
                 val availableSDKVersions = metadata.getJSONArray("availableSDKVersions")
                 val sdkVersionRequired = availableSDKVersions.getString(0)
                 formattedMessage =
-                  "This project uses SDK v" + sdkVersionRequired + " , but this version of Expo Go requires at least v" + Constants.SDK_VERSIONS_LIST[Constants.SDK_VERSIONS_LIST.size - 1] + "."
+                  "This project uses SDK v" + sdkVersionRequired + " , but this version of Expo Go requires at least v" + Constants.SDK_VERSIONS_LIST[Constants.SDK_VERSIONS_LIST.size - 1] + ". The author needs to update their project to a newer Expo SDK version."
```

Comment:
```suggestion
                  "This project uses SDK v" + sdkVersionRequired + " , but this version of Expo Go requires at least v" + Constants.SDK_VERSIONS_LIST[Constants.SDK_VERSIONS_LIST.size - 1] + ". The author needs to update their project to a newer Expo SDK version."
```

i think we should iterate over the list of supported sdk versions. 

> but this version of Expo Go supports SDK 40, 41, 42, and 43.

also would get rid of the "v" in front of SDK when we reference it, since we don't do that elswhere.

> This project uses SDK 39

rather than

> This project uses SDK v39